src/openrct2/interface/Colour.h

```diff
@@ -154,4 +154,6 @@ void colours_init_maps();
 uint8 blendColours(const uint8 paletteIndex1, const uint8 paletteIndex2);
 #endif
 
+uint8 findClosestPaletteIndex(uint8 red, uint8 green, uint8 blue);
```

Comment:
The point of a commit message is to tell why you did this.

Reply:
I believe this was done to expose the function for Nick and Jaxan's upcoming minimap feature. See Gitter.

Reply:
My point is that the commit message/description should make that clear.

Reply:
I don't disagree. You could even make the argument that the change can be introduced at a later point, e.g. in the eventual PR that makes it relevant.

Reply:
It's to make upcoming PRs smaller, yeah. I didn't see any harm in exposing it. It's a useful function? I dunno.. seemed like a small thing. I can delay it.